### PR TITLE
CONCF-1004 | add load more button

### PR DESCRIPTION
**Pull request by hakubo**

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by rogatty** [comment]
@hakubo please resolve conflicts

File: front/scripts/main/components/CuratedContentEditorImageSearchComponent.ts [hunk 1]
```diff
@@ -38,58 +38,74 @@ App.CuratedContentEditorImageSearchComponent = Em.Component.extend(
 		imageSize: 200,
 		searchLimit: 24,
 		nextBatch: 1,
+		batches: 1,
+		images: [],
+
+		hasNextBatch: Em.computed('batches', 'nextBatch', function() {
```
Comment (rogatty) [inline front/scripts/main/components/CuratedContentEditorImageSearchComponent.ts]: `: boolean`

File: front/scripts/main/components/CuratedContentEditorImageSearchComponent.ts [hunk 2]
```diff
@@ -38,58 +38,74 @@ App.CuratedContentEditorImageSearchComponent = Em.Component.extend(
...
-					dataType: 'json',
-					success: (data: SearchPhotoResponseInterface): void => {
-						resolve(data);
-					},
-					error: (data: any): void => {
-						reject(data);
-					}
-				});
-			});
+			return Em.$.getJSON(
```
Comment (rogatty) [inline front/scripts/main/components/CuratedContentEditorImageSearchComponent.ts]: :no_good:
Use model for that.

File: front/templates/main/components/curated-content-editor-image-search.hbs [hunk 3]
```diff
@@ -17,5 +17,7 @@
 		{{/each}}
 		{{loading-spinner active=isLoading overlay=false}}
 	</div>
+	{{!-- @TODO CONCF-956 add translations to Create new item --}}
```
Comment (rogatty) [inline front/templates/main/components/curated-content-editor-image-search.hbs]: `Load More`

File: front/templates/main/components/curated-content-editor-image-search.hbs [hunk 4]
```diff
@@ -17,5 +17,7 @@
...
+	<button {{action 'loadMore'}} class="{{if hasNextBatch 'visible'}}">Load More</button>
```
Comment (rogatty) [inline front/templates/main/components/curated-content-editor-image-search.hbs]: `{{i18n 'app:curated-content-load-more-items'}}`

**Pull request merged**